release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2893,6 +2893,18 @@ This issue affects the CPU load balancing features because these features depend
 
 * Broadcom network interface controllers in legacy Single Root I/O Virtualization (SR-IOV) do not support quality of service (QoS) and tag protocol identifier (TPID) settings for the SRIOV VLAN. This affects Broadcom BCM57414, Broadcom BCM57508, and Broadcom BCM57504. (link:https://issues.redhat.com/browse/RHEL-9881[*RHEL-9881*])
 
+* In hosted control planes for {product-title}, the following Operators and components are not tested (link:https://issues.redhat.com/browse/OCPSTRAT-605[*OCPSTRAT-605*]):
+
+** Performance Addon Operator
```

Comment:
Consider verifying with the SME if this should be Node Tuning Operator (NTO) instead of  the Performance Addon Operator (PAO) since in OCP 4.11, [PAO functions moved to NTO](https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-PAO-to-NTO).

Reply:
Good question, @abhatt-rh. This reference to Performance Add Operator is correct -- it should not be changed to NTO.